### DISCUSSION TITLE
Debugger: should `--inspect-wait` be default?

### DISCUSSION THREAD

**Discussion by Leokuma** [denoland/vscode_deno#772] [Ideas]
With [this addition](https://github.com/denoland/deno/pull/17001), I wonder if it would make sense to make `--inspect-wait` default instead of `--inspect` when debugging so that no breakpoints are ever missed.

**Comment by sant123** [comment]
This makes sense, haven't got a specific situation to use `--inspect` instead of `--inspect-wait`. The debugger experience have improved a lot because of this addition.

**Comment by bartlomieju** [comment]
FYI this has been implemented and will be released in `v3.16.0` of the extension, this week.

**Reply by sant123** [reply]
Awesome, thank you so much 😁
